device-types/Synology/DS1512+.yaml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: Synology
+model: DS1512+
+slug: synology-ds1512-plus
+part_number: DS1512+
+u_height: 4
+comments: '[Synology DS1512+ Specs](https://www.synology.com/en-us/products/DS1512+#specs)'
+weight: 4.25
+weight_unit: kg
+power-ports:
```

Comment:
To make sure we have the most complete information possible please add the `airflow` parameter.